README.md

```diff
@@ -1,6 +1,7 @@
 <p align="center">
-    <img src="askbuddie-bot.png" width="300">
-    </a>
+     <a href="https://www.askbuddie.com">
+         <img "askbuddie-bot.png" align="center" alt="askbuddie-icon" width="300"/>
```

Comment:
You are missing `src` attribute.

`<img "askbuddie-bot.png"` should be `<img src="askbuddie-bot.png"`.

Reply:
Updated to 
`<img src="askbuddie-bot.png"`